src/test/codewhisperer/service/securityIssueHoverProvider.test.ts

```diff
@@ -99,5 +102,81 @@ describe('securityIssueHoverProvider', () => {
             const actual = securityIssueHoverProvider.provideHover(mockDocument, new vscode.Position(2, 0), token.token)
             assert.strictEqual(actual.contents.length, 0)
         })
+
+        it('should remove hover if document changed on the same line', () => {
```

Comment:
When naming these the language should be more direct

```suggestion
        it('removes the issue hover if the document change on the same line', () => {
```